build.fsx

```diff
@@ -843,7 +840,7 @@ Target.create "CacheDotNetReleases" (fun _ ->
         |> Async.AwaitTask
         |> Async.RunSynchronously
         |> List.ofSeq
-        |> List.find (fun product -> product.ProductVersion.Equals("6.0"))
+        |> List.find (fun product -> product.ProductVersion.Equals("8.0"))
```

Comment:
I'm not sure what the logic for this (or cachedDotnetSdkReleases.json) should be when updating .NET versions. 
The current cachedDotnetSdkReleases.json only contains the .NET 6 releases and that seemed to cause test failures when built as .NET 8 and looking for the v8 SDK versions.
Changing this to 8.0 fixed that, but that might then not be right if fake-cli is multi-targetted at .NET 6 and .NET 8 as the .NET 6 version might still need the list of v6 releases?